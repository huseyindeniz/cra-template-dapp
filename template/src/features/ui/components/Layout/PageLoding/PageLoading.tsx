import { Box, VStack, Progress } from '@chakra-ui/react';
import { useTranslation } from 'react-i18next';

export const PageLoading: React.FC = () => {
  const { t } = useTranslation('Layout');
  return (
    <VStack mt="10">
      <Box fontSize={'xs'}>
        {t('Please wait while the requested page is loading...')}
      </Box>
      <Box w="sm">
        <Progress size="xs" isIndeterminate />
      </Box>
    </VStack>
  );
};
