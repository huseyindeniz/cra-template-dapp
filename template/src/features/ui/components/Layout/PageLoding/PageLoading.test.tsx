import { composeStories } from '@storybook/testing-react';
import { render } from '@testing-library/react';

import * as stories from './PageLoading.stories';

describe('Feature: UI', () => {
  describe('Component: Layout/Footer', () => {
    const { Default } = composeStories(stories);
    describe('Scenario: Error', () => {
      it('should render correctly', () => {
        // Arrange
        const { asFragment } = render(<Default {...Default.args} />);
        // Assert
        expect(asFragment).toMatchSnapshot();
      });
    });
  });
});
