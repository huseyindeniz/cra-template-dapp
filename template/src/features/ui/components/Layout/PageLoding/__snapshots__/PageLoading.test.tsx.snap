// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature: UI Component: Layout/Footer Scenario: Error should render correctly 1`] = `[Function]`;
